File: src/js/src/scopes/UnorderedListScope.js
function UnorderedListScope() {
	this.type = ScopeTypeCache.unorderedList;
	this.level = ScopeLevel.block;
	this.open = function(analyzer) {
		analyzer.openUnorderedList();
	};
	this.close = function(analyzer) {
		analyzer.closeUnorderedList();
	};
}

extend(Scope, UnorderedListScope);

File: src/dotnet/Src/ButterflyNet.Parser/Scopes/UnorderedListScope.cs
namespace ButterflyNet.Parser.Scopes {
	public class UnorderedListScope : ListScope {
		public override void Open(ButterflyAnalyzer analyzer) {
			analyzer.OpenUnorderedList();
		}

		public override void Close(ButterflyAnalyzer analyzer) {
			analyzer.CloseUnorderedList();
		}
	}
}